Author: Andreas Beckmann <debian@abeckmann.de>
Description: Conditionally include linux/version.h
 Starting with 195.xx the include order in nv-linux.h was changed and
 therefore our conftest.h needs to include linux/version.h itself.
 Since linux/version.h does not contain include guards, add a check to
 nv-linux.h to not include linux/version.h multiple times.

Index: nvidia-173.14.25/nv-linux.h
===================================================================
--- nvidia-173.14.25.orig/nv-linux.h	2010-01-28 11:35:34.000000000 +0100
+++ nvidia-173.14.25/nv-linux.h	2010-06-10 13:22:44.416037156 +0200
@@ -21,7 +21,9 @@
 #include <linux/autoconf.h>
 #endif
 
+#ifndef LINUX_VERSION_CODE
 #include <linux/version.h>
+#endif
 #include <linux/utsname.h>
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2, 4, 7)
